api/routes/api.php

```diff
@@ -28,6 +29,8 @@
     Route::apiResource('/role', RoleController::class);
     Route::apiResource('/department', DepartmentController::class);
     Route::apiResource('/announcement', AnnouncementController::class);
+    Route::get('/thread/{id}', [ThreadController::class, 'fetchThread']);
```

Comment:
We can group this one with route prefixes sir if you want to.
![image](https://user-images.githubusercontent.com/109492180/194797678-7fc8955e-8b0c-42c6-8939-4ac655a12c7b.png)